images/plbase/python-requirements.txt

```diff
@@ -1,17 +1,17 @@
-ansi2html==1.9.1
+ansi2html==1.9.2
 chevron==0.14.0
 coloraide==3.3.1
-Faker==25.3.0
+Faker==26.0.0
 lxml==5.2.2
 matplotlib==3.9.0
 networkx==3.3
 nltk==3.8.1
 numpy==1.26.4
-openpyxl==3.1.3
-pandas-stubs==2.2.2.240514
+openpyxl==3.1.5
+pandas-stubs==2.2.2.240603
 pandas==2.2.2
-Pint==0.23
-psutil==5.9.8
+Pint==0.24.1
+psutil==6.0.0
```

Comment:
All bugfixes in this major version bump: https://github.com/giampaolo/psutil/blob/master/HISTORY.rst